code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -122,7 +122,7 @@ paths:
         '401':
           $ref: '#/components/responses/Unauthenticated'
         '403':
-          $ref: '#/components/responses/NotAuthenticatedByIpPermissionDenied'
+          $ref: '#/components/responses/PhoneNumberVerificationPermissionDenied'
```

Comment:
The more generic error descriptions are there, the better, so this change is good.